tests/test_nan.py

```diff
@@ -12,36 +11,41 @@ class NanTest(dj.Manual):
     """
 
 
-@pytest.fixture(scope="module")
-def schema(connection_test):
```

Comment:
Trying to give these fixtures more descriptive and unique names. While reusing the name `schema` between test modules technically works, unique names like `schema_nan` make it clearer that tests in different modules are using entirely different fixtures.